src/content/docs/accounts/accounts/account-maintenance/promotions-ui.mdx

```diff
@@ -0,0 +1,108 @@
+* Enrollment flexibility: Easily sign-up or leave from product previews until they become generally available (GA)
+* Upgrade options: Upgrade to access products once they are generally available 
+
+To help you get started, we've provided some tips about eligibility and enrollment.
+
+## Eligibility [#eligibility]
+
+You can check the plan management page to see if your organization is eligible for a product promotion. If you don't see the promotions UI, your organization isn't eligible.
+
+All promotions apply to the entire organization, including all users and all accounts that belong to your organization.
+
+### Users [#users]
+
+Any user in an organization can sign up for a product promotion. However, only users who have the **Billing** administrative setting in their group can do the following: 
+
+* Leave a promotion 
+* Re-enroll after leaving a promotion
+
+### Organizations [#org-rules]
+
+Organizations can be enrolled in more than one preview at a time. Also, most organizations are eligible for promotions, with some exceptions. Ineligible organizations are ones that: 
+
+* Have HIPAA-enabled accounts
+* Use FedRAMP-compliant endpoints
+* Don't allow for click-through agreement (for example, key accounts) 
+* Have contractual restrictions
+* Use the original product-based pricing model
+
+## Browse available previews [#access-ui]
+
+The promotions management UI lists available product promotions in a single, consolidated table view. Each row in the table contains essential information such as the product name, a brief description, preview status (**Available**, **Opted In**, **Opted Out**), and the anticipated end date of the preview. 
+
+    <img
+        title="Screenshot showing the promotions table"
+        alt="Screenshot showing the promotions table"
+        src={accountsPromotionsTable}
+    />
+
+Some promotions don't have a defined end date, so you may see the end date as **Ongoing**. We try to provide a best estimate of the length of the preview, but it is provided for informational purposes only.
+
+<Callout variant="important">
+  The end date should not be construed as an offer, commitment, promise or obligation on behalf of New Relic to deliver any product, material, code, functionality, or other feature.
+</Callout>
+
+## Sign up for a product promotion [#sign-up-or-leave]
+
+To sign up for a promotion or to leave one you're currently enrolled in: 
+
+1. Log into New Relic.
+2. From the [user menu](/docs/accounts/accounts-billing/general-account-settings/intro-account-settings), go to **Administration > Usage Summary**.
+3. Click **Previews & trials** (If you can't see this menu, you're not eligible):
+    <img
+        title="Screenshot showing the promotions UI"
+        alt="Screenshot showing the promotions UI"
+        src={accountsPreviewsandTrials}
+    />
+4. Open the ellipsis menu (**...**) for the promotion you're interested in, and then select one of the following:
+    * **Enroll**
+    * **Leave Preview**
```

Comment:
```suggestion
4. Open the ellipsis menu (**...**) for the promotion you're interested in, and then click **Enroll**.
```

Reply:
@ally-sassman I was trying to clarify that there are two options. Are you saying I should just focus on enrollment?